Update readme (#15)
lazybytez/jojo-discord-bot: README.md

# JOJO Discord Bot

  ![commit-info][commit-info]
  ![contributors-info][contributors-info]
  ![reposize-info][reposize-info]
  ![stars][stars]

## Description

> This open source Discord bot is mainly developed by lazybytez, but anyone who wants to contribute can join.  

Reviewer: ```suggestion
This is an open source Discord bot is mainly developed by Lazy Bytez.  
```

> If you want to help with the development of the bot please check out the [Contributing](https://github.com/lazybytez/jojo-discord-bot#contributing) section.  

Reviewer: ```suggestion
If you want to take place in the development of the bot please check out the [Contributing](https://github.com/lazybytez/jojo-discord-bot#contributing) section.  
```
